generate-templates.yaml

```diff
@@ -12,14 +12,14 @@
       src: rhel7.tpl.yaml
       dest: "{{ playbook_dir }}/dist/templates/{{ os }}-{{ item.workload }}-{{ item.flavor }}.yaml"
     with_items:
-    - {flavor: tiny, workload: generic, memsize: "1G", cores: 1, iothreads: False}
-    - {flavor: tiny, workload: highperformance, memsize: "1G", cores: 1, iothreads: True}
-    - {flavor: small, workload: generic, memsize: "2G", cores: 1, iothreads: False}
-    - {flavor: small, workload: highperformance, memsize: "2G", cores: 1, iothreads: True}
-    - {flavor: medium, workload: generic, memsize: "4G", cores: 1, iothreads: False}
-    - {flavor: medium, workload: highperformance, memsize: "4G", cores: 1, iothreads: True}
-    - {flavor: large, workload: generic, memsize: "8G", cores: 2, iothreads: False}
-    - {flavor: large, workload: highperformance, memsize: "8G", cores: 2, iothreads: True}
+    - {flavor: tiny, workload: generic, memsize: "1G", cores: 1, sockets: 1, threads: 1, iothreads: False}
+    - {flavor: tiny, workload: highperformance, memsize: "1G", cores: 1, sockets: 1, threads: 1, iothreads: True}
+    - {flavor: small, workload: generic, memsize: "2G", cores: 1, sockets: 1, threads: 1, iothreads: False}
+    - {flavor: small, workload: highperformance, memsize: "2G", cores: 1, sockets: 1, threads: 1, iothreads: True}
+    - {flavor: medium, workload: generic, memsize: "4G", cores: 1, sockets: 1, threads: 1, iothreads: False}
+    - {flavor: medium, workload: highperformance, memsize: "4G", cores: 1, sockets: 1, threads: 1, iothreads: True}
+    - {flavor: large, workload: generic, memsize: "8G", cores: 2, sockets: 1, threads: 1, iothreads: False}
+    - {flavor: large, workload: highperformance, memsize: "8G", cores: 2, sockets: 1, threads: 1, iothreads: True}
```

Comment:
Please change the configuration of all OSes so it uses sockets instead of cores.